indico/modules/events/editing/client/js/components/Timeline/judgment/UpdateFilesForm.jsx

```diff
@@ -8,11 +8,11 @@
 import uploadURL from 'indico-url:event_editing.api_upload';
 
 import _ from 'lodash';
-import React from 'react';
+import React, {useState, Fragment} from 'react';
```

Comment:
for fragment it's more common to write `<React.Fragment>` instead of importing `Fragment` itself